media/alexnet.prototxt

```diff
@@ -1,272 +0,0 @@
-name: "AlexNet"
```

Comment:
No, now we also have a copy because of the front-end model zoo so we can remove this